frontend/app/templates/components/user-component/my-badges.hbs

```diff
@@ -69,21 +71,24 @@
                 <input type="checkbox" class="ui checkbox downloadCheck" id={{badge.download_link}}>
                 {{/ui-popup}}
               </div>
-
             </div>
           </div>
         </div>
         {{/each}}
-        {{/if}}
         <div class="card">
           <a class="" href="/create-badges">
-            {{#if model.length}}
             <img src="/images/add_more.png" style="margin-top: 70px">
-            {{else}}
-            <img src="/images/add_more.png" style="margin-top: 70px; margin-bottom: 90px">
-            {{/if}}
           </a>
         </div>
+        {{else}}
+        <center>
+          <h2>You have'nt created any badges yet!</h2>
+          <img src="/images/empty.png" style="width:30%;">
+          <a class="" href="/create-badges">
```

Comment:
I think both the lines can be merged , Since you are making the project alive , remove duplication anywhere you see

Reply:
Inline would make the line greater than 80. It's a link so merging can't be done unless with js

Reply:
@abishekvashok I was saying use this 
```
<a class="ui fluid submit orange basic button" href="/create-badges" style="width: 30%">Create Badge</a>

Reply:
Cool